Gemfile

```diff
@@ -7,3 +7,4 @@ gemspec
 # gem 'byebug', group: [:development, :test]
 
 gem 'workarea', github: 'workarea-commerce/workarea'
+gem 'workarea-search_autocomplete', github: 'workarea-commerce/workarea-search-autocomplete', branch: 'bugfix/SEARCHAUTO-1'
```

Comment:
waiting on workarea-commerce/workarea-search-autocomplete#5 to merge, then I can change this to `branch: :master`